package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-chartutils",
-  "version": "6.0.4",
+  "version": "6.0.5",
```

Comment:
Let's increase the version to 7.0.0

Reply:
fixed 1aad98d1d3240cd7650254b1610f2b4f007af271